package.json

```diff
@@ -34,7 +34,7 @@
         "@angular/platform-browser": "^15.1.0",
         "@angular/platform-browser-dynamic": "^15.1.0",
         "@angular/router": "^15.1.0",
-        "@bananocoin/bananojs-hw": "^1.5.4",
+        "@bananocoin/bananojs-hw": "../bananojs-hw",
```

Comment:
This should be changed back to the online repository once this pull-request is merged and published:
https://github.com/BananoCoin/bananojs-hw/pull/2